US-Trustee-Program/Bankruptcy-Oversight-Support-Systems
# 111 - Improvements to Bicep deployment scripts

Description of changes
- Reorganize existing bicep files
- Leverage modules
- Add support to create/update resources that may belong to different Resource Groups
- Add bash scripts for deployments and replace steps and inline scripts in deployment
- Update deployment workflow by separating webapp and backend deployment to separate job
- Bicep updates to enable Azure functions for Java and Node runtime environments
- Update webapp and functionapp deploy to limit access to only the build agent during deployment

Validation
- Verify full creation of resources in Azure environment
- Verify deployment is repeatable
- Verify deployment for Java and Node functions
